### PR TITLE
typo and code is similar to math in linear_algebra, orth_proj, and lln_clt.

### PR DESCRIPTION
**Pull request by PooyaFa**
Previous commits also consist of replacing bellman_operator with T.

### CODE REVIEW

**Comment by PooyaFa** [comment]
@arnavs  Please, notice that I pushed another commit to Pooya1 and you can see. Thank you!

**Comment by PooyaFa** [comment]
> Hey @PooyaFa, should I close the other PR in favor of this one?

Yes thank you. I will apply your suggestions, and will update this branch.

**Comment by PooyaFa** [comment]
I will also, run and test them again, so we can be sure that everything works fine. I will commit on this branch very soon.

**Comment by arnavs** [comment]
OK great, thanks. 

For the tests, just to confirm, switch into the branch, `make jupyter-test` (or `jupyter-tests` or whatever we decided to call it, you can check the `Makefile`). Then `run all cells` in the relevant notebook, and you can just do a control + f for `@testset` and make sure that all of them pass.

**Comment by PooyaFa** [comment]
@arnavs Hi. I fixed the mistakes and test all the files. Thanks.

**Comment by arnavs** [comment]
Thanks @PooyaFa, this looks much better.

File: rst_files/odu.rst [hunk 1]
```diff
@@ -329,7 +329,7 @@ The code is as follows
 
 The type ``SearchProblem`` is used to store parameters and methods needed to compute optimal actions
 
-The Bellman operator is implemented as the method ``.bellman_operator()``, while ``.get_greedy()``
+The Bellman operator is implemented as the method ``.T()``, while ``.get_greedy()``
```
Comment (arnavs) [inline rst_files/odu.rst]: Removing the dots as they probably come from Python. 

```suggestion
The Bellman operator is implemented as the method ``T()``, while ``get_greedy()``
```

**Pull request merged**